bin/deploy-vhost

```diff
@@ -1127,17 +1081,26 @@ sub pgpw {
     return $_;
 }
 
-sub chdir_verbose {
-    my ($line, $dir) = @_;
-    if ($verbose) {
-        print STDERR "Changing directory (at line $line) to $dir\n";
+sub mugly {
+    my ($in, $out, $extra_settings) = @_;
+    my @args = ("$mysociety_bin/mugly", "-O", $out, "-p", $settings_file;
```

Comment:
Missing closing parenthesis, but also shouldn't it be `my @args = ("$mysociety_bin/mugly", "-O", $out);` anyway as you `push` the optional arguments on in the next line if necessary?

Reply:
Whoops!
You always have `settings_file`, you might also have `extra_settings` as *another* `-p` entry.